### PR TITLE
[fastboot] fastboot enhancement: use warm-reboot infra

### PR DESCRIPTION
**Pull request by arfeigin**
Improve fastboot by using warmboot infrastructure.
Change warm_restart to advanced_restart.
Replace WARM tables with ADVANCED tables on all files.
Add rename function to redis interface.

### CODE REVIEW

File: common/dbinterface.cpp [hunk 1]
```diff
@@ -72,6 +72,17 @@ std::shared_ptr<std::string> DBInterface::get(const std::string& dbName, const s
     return blockable<std::shared_ptr<std::string>>(innerfunc, dbName, blocking);
 }
 
+bool DBInterface::rename(const std::string& dbName, const std::string& hash, const std::string& newHash, bool blocking)
```
Comment (qiluo-msft) [inline common/dbinterface.cpp:75]: >, bool blocking [](http://example.com/codeflow?start=103&length=15)

You can remove this parameter.

File: common/schema.h [hunk 2]
```diff
@@ -315,7 +315,7 @@ namespace swss {
 
 #define CFG_POLICER_TABLE_NAME                      "POLICER"
 
-#define CFG_WARM_RESTART_TABLE_NAME                 "WARM_RESTART"
```
Comment (qiluo-msft) [inline common/schema.h:318]: >WARM_RESTART [](http://example.com/codeflow?start=53&length=12)

config_db.json will be carried over to new image, so we normally need backward-compatible for ConfigDB schema.

File: common/sonicv2connector.cpp [hunk 3]
```diff
@@ -89,6 +89,11 @@ std::map<std::string, std::string> SonicV2Connector_Native::get_all(const std::s
     return m_dbintf.get_all(db_name, _hash, blocking);
 }
 
+bool SonicV2Connector_Native::rename(const std::string& db_name, const std::string& _hash, const std::string& newHash, bool blocking)
```
Comment (qiluo-msft) [inline common/sonicv2connector.cpp:92]: >blocking [](http://example.com/codeflow?start=124&length=8)

You can remove this paramter

File: common/dbconnector.cpp [hunk 4]
```diff
@@ -690,6 +690,15 @@ bool DBConnector::flushdb()
     return s == "OK";
 }
 
+bool DBConnector::rename(const string &key, const string &newKey)
```
Comment (qiluo-msft) [inline common/dbconnector.cpp:693]: >rename [](http://example.com/codeflow?start=18&length=6)

Suggest extrace `rename` related to a standalone PR. Please add unit test.

**Comment by arfeigin** [comment]
@qiluo-msft  thanks for reviewing the changes. After a meeting with @vaibhavhd , @prsunny , and @yxieca we are rolling back the suggested changes - a new set of PRs will be ready in few days.
Thanks for your time.

**Pull request closed without merging**